Cython/Compiler/Nodes.py

```diff
@@ -9090,6 +9090,7 @@ class ParallelStatNode(StatNode, ParallelNode):
 
     num_threads = None
     chunksize = None
+    _if = None
```

Comment:
```suggestion
    if_ = None
```

I think I prefer `if_` anyway

Reply:
It's a typo thanks!